src/constants/CardPropTypes.js

```diff
@@ -413,6 +413,11 @@ export const CardPropTypes = {
   isEditable: PropTypes.bool,
   /** goes full screen if expanded */
   isExpanded: PropTypes.bool,
+  /**
+   * Define the icon render to be rendered.
+   * Can be a React component class
```

Comment:
This comment is a bit confusing, does it have to be a react **class**?

Reply:
This is the same comment that appears in Button.jsx in the carbon library, keeping it the same to avoid confusion for passing in renderIcons